### PR TITLE
adds create_table, create_dataset, and copy_table to bigquery samples

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: bigquery/api/src/CopyTableCommand.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (!$projectId = $input->getOption('project')) {
+            $projectId = $this->getProjectIdFromGcloud();
+        }
+        $datasetId = $input->getArgument('dataset');
+        $sourceTableId = $input->getArgument('source-table');
+        $builder = new ServiceBuilder([
```
Comment (tmatsuo) [inline bigquery/api/src/CopyTableCommand.php]: Do you want to keep using the `ServiceBuilder`?

Comment (bshaffer) [inline bigquery/api/src/CopyTableCommand.php]: no, I'll update all the samples :D

Comment (bshaffer) [inline bigquery/api/src/CopyTableCommand.php]: On second thought, this should be done in another PR, because if we change it for this sample we should change it for all of them, to be consistent.

Comment (bshaffer) [inline bigquery/api/src/CopyTableCommand.php]: see https://github.com/GoogleCloudPlatform/php-docs-samples/pull/237
